### ISSUE TITLE
Test: Add regression test for `updateStakes` infinite loop

### ISSUE THREAD

**Issue by wadealexc** [Layr-Labs/eigenlayer-middleware#24]
PR here: https://github.com/Layr-Labs/eigenlayer-middleware/pull/23

**Comment by ChaoticWalrus** [comment]
Looking into this now

**Comment by ChaoticWalrus** [comment]
I think that a regression test for this particular issue no longer makes sense, due to changes in the relevant functions.
The `StakeRegistry.updateStakes` function morphed into the `StakeRegistry.updateOperatorStake` function, but it is now callable only by the RegistryCoordinator https://github.com/Layr-Labs/eigenlayer-middleware/blob/m2-mainnet/src/StakeRegistry.sol#L147-L189 -- which is currently only done within the `RegistryCoordinator.updateOperators` function.
In short, this functionality has structurally changed enough that I don't even know what a regression test would look like any more. We should however make sure that the `StakeRegistry.updateStakes` function performs proper input validation.
